src/plugins/intel_gpu/src/kernel_selector/cl_kernels/grid_sample_ref.cl

```diff
@@ -1,40 +1,240 @@
-// Copyright (C) 2022 Intel Corporation
```

Comment:
@ozhydkov-lohika Please add back the copyright.

Reply:
resolve